js/client/modules/@arangodb/testutils/instance-manager.js

```diff
@@ -524,20 +527,48 @@ class instanceManager {
   // / @brief dump the state of the agency to disk. if we still can get one.
   // //////////////////////////////////////////////////////////////////////////////
   dumpAgency() {
+    const dumpdir = fs.join(this.options.testOutputDirectory, `agencydump_${this.instanceCount}`);
+    const zipfn = fs.join(this.options.testOutputDirectory, `agencydump_${this.instanceCount}.zip`);
+    if (fs.isFile(zipfn)) {
+      fs.remove(zipfn);
+    };
+    if (fs.exists(dumpdir)) {
+      fs.list(dumpdir).forEach(file => {
+        const fn = fs.join(dumpdir, file);
+        if (fs.isFile(fn)) {
+          fs.remove(fn);
+        }
+      });
```

Comment:
Why we can't just remove all dumpdir without iterating over it? 
Why we delete here only files but not directories?

May be we can use something like this?
```
fs.rmSync(dir, { recursive: true, force: true });
```

Reply:
there may be zips and other json reports in this directory,

Reply:
So we will delete only files on the first leve of `dumpdir`, right? Directories will be not processed?